pallets/attestation/src/lib.rs

```diff
@@ -123,7 +123,7 @@ pub mod pallet {
 		type WeightInfo: WeightInfo;
 
 		/// The currency that is used to reserve funds for each attestation.
-		type Currency: Currency<AccountIdOf<Self>> + ReservableCurrency<AccountIdOf<Self>>;
+		type Currency: ReservableCurrency<AccountIdOf<Self>>;
```

Comment:
Definitely nice catch! As a note: In general, we try to not sneak changes independent of the task into our PRs.

Reply:
Normally I also wouldn't have included it, but I felt encouraged because of the `I have left the code in a better state` bullet point in the PR description template.